.pullapprove.yml

```diff
@@ -48,8 +48,7 @@ groups:
           - "*.md"
         exclude:
           - "vendor/*"
-          - "cmd/fetchbranches/vendor/*"
-          - "cmd/localCI/vendor/*"
+          - "*/vendor/*"
```

Comment:
Nit: would be good to have this next to the other vendor exclude. In fact, as this file is yaml, it should be possible to add a comment in it explaining why we have multiple vendor excludes, but I don't believe anyone has tested comments in `.pullapprove.yml` yet ;)